client/i18n/locales/english/intro.json

```diff
@@ -1882,6 +1882,7 @@
       "hoec": { "title": "136", "intro": [] },
       "cygu": { "title": "137", "intro": [] },
       "axgb": { "title": "138", "intro": [] },
+      "nnqa": { "title": "154", "intro": [] },
```

Comment:
this could be deleted right?

Reply:
Yes, this shouldn't be here. I have removed it.